src/BuildServer/BuildServerResponse.cs

```diff
@@ -0,0 +1,96 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+
+namespace Microsoft.Oryx.BuildServer
+{
+    enum BuildState
+    {
+        Success,
+        Running,
```

Comment:
Nit: Maybe we should consider to rename to `Building`. Running sounds like it like the built app is running to me.

Reply:
I like Building .. good one